manifests/charts/ztunnel/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if (.Values.global.networkPolicy).enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ztunnel
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: ztunnel
+    istio.io/rev: {{ .Values.revision | default "default" | quote }}
+    install.operator.istio.io/owning-resource: {{ .Values.ownerName | default "unknown" }}
+    app.kubernetes.io/name: ztunnel
+    operator.istio.io/component: "Ztunnel"
+    {{- include "istio.labels" . | nindent 4 }}
+spec:
+  podSelector:
+    matchLabels:
+      app: ztunnel
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
```

Comment:
not a ztunnel expert, but from[ architecture docs](https://github.com/istio/ztunnel/blob/2837b0f410d8e026969900b00e7fa5c685187873/ARCHITECTURE.md?plain=1#L15)

looks like may need to add 15008/15006.